src/sentry/stacktraces/processing.py

```diff
@@ -489,6 +490,35 @@ def dedup_errors(errors):
     return rv
 
 
+def suspected_console_errors(mechanism, error_type, frames):
+    def is_suspicious_frame(frame) -> bool:
```

Comment:
I'd recommend wrapping this method in a try/catch and returning false on an error. We should be careful here since this is part of event processing, and if we introduce a bug it'll cause an outage.

Reply:
Good call, done. Handling the frame evaluation and the tagging as separate try/excepts.